src/ui/component/header/view.jsx

```diff
@@ -37,10 +37,12 @@ const Header = (props: Props) => {
     automaticDarkModeEnabled,
     hideBalance,
     email,
-    minimal,
+    authHeader,
     signOut,
   } = props;
   const authenticated = Boolean(email);
+  const homeButtonNavigationProps = authHeader ? { onClick: signOut } : { navigate: '/' };
+  const closeButtonNavigationProps = authHeader ? { onClick: signOut } : { onClick: () => history.goBack() };
```

Comment:
I think it makes more sense to just log out and stay on the auth page, than redirect back to the page they were on. I think in most cases they will want to log back in, and when they do, the redirect will still exist to go back to the page they were on.

Reply:
Agreed that back is not correct for sign out. I think homepage is most common behavior on sign out.